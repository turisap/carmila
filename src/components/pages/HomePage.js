/**
 * Created by HP on 09-Dec-17.
 */
import React from 'react';


const HomePage = () => (
<div>
    <section id="bestFood">
        <div className="container-fluid bestFood__container">
            <div className="row">
                <div className="col-md-6 offset-md-3">
                    <h1 className="text-center bestFood__title">Find the best food in</h1>
                    <h1 className="text-center bestFood__title">Battambang</h1>
                </div>
            </div>
        </div>
    </section>
    <section id="homePagePhotos">
        <div className="container-fluid">
            <div className="row">
                <div className="col-md-4">
                    <img className="homePagePhotos_image homePagePhotos_main" src={"./img/pages/fire.jpg"}/>
                </div>
                <div className="col-md-4">
                    <div className="homePagePhotos_cropper">
                        <img className="homePagePhotos_image"  src={"./img/pages/dessert.jpg"}/>
                        <img className="homePagePhotos_image"  src={"./img/pages/pancakes.jpg"}/>
                    </div>
                </div>
                <div className="col-md-4">
                    <div className="homePagePhotos_cropper">
                        <img className="homePagePhotos_image"  src={"./img/pages/pizza_home.jpg"}/>
                        <img className="homePagePhotos_image"  src={"./img/pages/salami.jpg"}/>
                    </div>
                </div>
            </div>
        </div>
    </section>
</div>
);

export default HomePage;