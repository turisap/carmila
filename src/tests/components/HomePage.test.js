/**
 * Created by HP on 15-Dec-17.
 */
import React from 'react';
import { shallow } from 'enzyme';
import HomePage from '../../components/pages/HomePage';

test('should render homepage properly', () => {
    const wrapper = shallow(<HomePage/>);
    expect(wrapper).toMatchSnapshot();
});