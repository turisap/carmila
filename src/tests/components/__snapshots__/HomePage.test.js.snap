// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render homepage properly 1`] = `
<div>
  <section
    id="bestFood"
  >
    <h1
      className="text-center bestFood__title"
    >
      Find the best food in
    </h1>
    <h1
      className="text-center bestFood__title"
    >
      Battambang
    </h1>
  </section>
  <section
    id="intro"
  >
    <div
      className="container-fluid"
    >
      <div
        className="row"
      >
        <div
          className="col-md-8 offset-md-2"
        >
          <p
            className="text-center intro__text"
          >
            We are a group of small take-away European restaurants in the Battambang province and offer you a nice opportunity to try various items from Italian, Portuguese German cuisine. We have five take-away points in the city and another one in the Veren village. We work 24/7 to ensure you always can enjoy delicious food.
          </p>
        </div>
      </div>
    </div>
  </section>
  <section
    id="homePagePhotos"
  >
    <div
      className="container-fluid"
    >
      <div
        className="row"
      >
        <div
          className="col-4 homePagePhotos__column"
        >
          <img
            className="homePagePhotos__image "
            src="./img/pages/fire.jpg"
          />
        </div>
        <div
          className="col-4 homePagePhotos__column"
        >
          <div
            className="homePagePhotos__cropper"
          >
            <img
              className="homePagePhotos__image"
              src="./img/pages/dessert.jpg"
            />
            <img
              className="homePagePhotos__image"
              src="./img/pages/pancakes.jpg"
            />
          </div>
        </div>
        <div
          className="col-4 homePagePhotos__column"
        >
          <div
            className="homePagePhotos__cropper"
          >
            <img
              className="homePagePhotos__image"
              src="./img/pages/pizza_home.jpg"
            />
            <img
              className="homePagePhotos__image"
              src="./img/pages/salami.jpg"
            />
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;
